test/unit/api_docs/provider_data_test.rb

```diff
@@ -0,0 +1,19 @@
+require 'test_helper'
+
+class ApiDocs::ProviderDataTest < ActiveSupport::TestCase
+  def setup
+    @account = FactoryBot.create(:simple_provider)
+  end
+
+  attr_accessor :account
+
+  def test_provider_users_ids
+    FactoryBot.create(:simple_user, username: 'admin1', role: :admin, account: account)
+    FactoryBot.create(:simple_user, username: 'admin2', role: :admin, account: account)
+    FactoryBot.create(:simple_user, username: 'member1', role: :member, account: account)
+    FactoryBot.create(:simple_user, username: 'member2', role: :member, account: account)
```

Comment:
maybe create more users to make sure last 5 only are returned. You can use `#create_list`